app/views/namespaces/_namespace.html.slim

```diff
@@ -1,6 +1,7 @@
 tr id="namespace_#{namespace.id}"
   td= link_to namespace.clean_name, namespace
   td= namespace.repositories.count
+  td= namespace.webhooks.count
```

Comment:
Should this link replace the existing "Show webhooks" link on the namespace page (`app/views/namespaces/show.html.slim`), or should they coexist?